src/components/dataDisplays/cellRenderers/SocialGroupRoleRenderer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { get } from 'lodash-es';
+
+import useSiteSettings from '../../../models/site/useSiteSettings';
+import Text from '../../Text';
+import OverflowController from './OverflowController';
+
+export default function SocialGroupRoleRenderer({
+  value,
+  noWrap = false,
+  ...rest
+}) {
+  const intl = useIntl();
+
+  const { data } = useSiteSettings();
+  const roles = get(data, ['social_group_roles', 'value'], []);
+  const matchingRole = roles.find(role => role?.guid === value);
```

Comment:
Because the native one works fine here, as far as I'm aware. Cool thing about the lodash one is you can use it on objects but here we just have an array